common/src/main/java/com/wynntils/features/stateless/ChatItemFeature.java

```diff
@@ -22,19 +22,15 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import org.lwjgl.glfw.GLFW;
 
-public class ChatItemFeature extends InternalFeature {
+public class ChatItemFeature extends StatelessFeature {
     private final Map<String, String> chatItems = new HashMap<>();
 
     @Override
-    protected void onInit(ImmutableList.Builder<Condition> conditions) {
+    protected void onInit(
+            ImmutableList.Builder<Condition> conditions, ImmutableList.Builder<Class<? extends Manager>> dependencies) {
         conditions.add(new WebLoadedCondition());
     }
 
-    @Override
-    protected boolean onEnable() {
-        return WebManager.isItemListLoaded() || WebManager.tryLoadItemList();
-    }
-
     @SubscribeEvent
```

Comment:
It's not at all clear to me why this is not a normal user enabled feature? We're doing a lot of slightly scary stuff here. Why should the user not be able to turn this off?

I'm thinking a reason for this might be that there is some kind of coupling/dependency between this feature and the screenshot feature. But then we should be able to express "recommends" or "depends" relationships between features instead.

Reply:
Iirc it should be fine ti disable this, only chat item parsing breaks.

Reply:
Done